### PR TITLE
added update-all-analyzers method to search

### PR DESCRIPTION
**Pull request by abner-dou**
## Description

Method to update all search analyzers was added

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by themantissa — APPROVED** [review]
LGTM, thank you!

**Review by navenoxin — APPROVED** [review]

**Review by threebee — APPROVED** [review]
LGTM!

File: mongodbatlas/search.go [hunk 1]
```diff
@@ -214,6 +215,31 @@ func (s *SearchServiceOp) ListAnalyzers(ctx context.Context, groupID, clusterNam
...
 }
 
+// UpdateAllAnalyzers Update All User-Defined Analyzers for a specific Cluster.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/fts-analyzers-update-all//
+func (s *SearchServiceOp) UpdateAllAnalyzers(ctx context.Context, groupID, clusterName string, analyzers []*SearchAnalyzer) ([]*SearchAnalyzer, *Response, error) {
+	if groupID == "" {
+		return nil, nil, NewArgError("groupID", "must be set")
+	}
+	if clusterName == "" {
```
Comment (thetonymaster) [inline mongodbatlas/search.go:225]: `if len(clusterName) == 0`

File: mongodbatlas/search.go [hunk 2]
```diff
@@ -214,6 +215,31 @@ func (s *SearchServiceOp) ListAnalyzers(ctx context.Context, groupID, clusterNam
 	return root, resp, err
 }
 
+// UpdateAllAnalyzers Update All User-Defined Analyzers for a specific Cluster.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/fts-analyzers-update-all//
+func (s *SearchServiceOp) UpdateAllAnalyzers(ctx context.Context, groupID, clusterName string, analyzers []*SearchAnalyzer) ([]*SearchAnalyzer, *Response, error) {
+	if groupID == "" {
```
Comment (thetonymaster) [inline mongodbatlas/search.go:222]: `if len(groupID ) == 0`

File: mongodbatlas/search.go [hunk 3]
```diff
@@ -214,6 +215,31 @@ func (s *SearchServiceOp) ListAnalyzers(ctx context.Context, groupID, clusterNam
...
+		return nil, nil, NewArgError("groupID", "must be set")
+	}
+	if clusterName == "" {
+		return nil, nil, NewArgError("clusterName", "must be set")
+	}
+
+	path := fmt.Sprintf(searchBasePath, groupID, clusterName)
+	path = fmt.Sprintf("%s/analyzers", path)
```
Comment (thetonymaster) [inline mongodbatlas/search.go:230]: I assume `searchBasePath` is a constant, so can be `%s/analyzers`

**Review by thetonymaster — APPROVED** [review]
Non blocking issues, just nitpicks

**Pull request merged**